CRM/Contribute/BAO/ContributionPage.php

```diff
@@ -356,7 +356,11 @@ public static function sendMail($contactID, $values, $isTest = FALSE, $returnMes
         );
       }
 
-      $title = isset($values['title']) ? $values['title'] : CRM_Contribute_PseudoConstant::contributionPage($values['contribution_page_id']);
+      $contribution_page_title = '';
```

Comment:
The line above doesn't assume contribution_page_id could be empty - & not assuming it tidies up this block a little. ie. you just need one line then